templates/kubernetes/terraform/environments/development/main.tf

```diff
@@ -1,15 +1,25 @@
+terraform {
```

Comment:
The backend block can't be in a module and can't contain string interpolation so I moved it in here.